input/resources/StructureDefinition-extension-qualification.json

```diff
@@ -0,0 +1,232 @@
+        "fixedUri": "https://build.fhir.org/ig/HL7/fhir-us-ndh/ValueSet-SpecialtyAndDegreeLicenseCertificateVS.html"
+      },
+      {
+        "id": "Extension.extension:issuer",
+        "path": "Extension.extension",
+        "sliceName": "issuer",
+        "min": 0
+      },
+      {
+        "id": "Extension.extension:issuer.url",
+        "path": "Extension.extension.url",
+        "fixedUri": "issuer"
+      },
+      {
+        "id": "Extension.extension:issuer.value[x]",
+        "path": "Extension.extension.value[x]",
+        "definition": "Reference(NDH Base Organization Profile | NDH Exchange Organization Profile)",
+        "comment": "Reference(NDH Base Organization Profile | NDH Exchange Organization Profile)",
+        "type": [
+          {
+            "code": "Reference"
+          }
+        ]
+      },
+      {
+        "id": "Extension.extension:Status",
+        "path": "Extension.extension",
+        "sliceName": "Status",
+        "min": 1,
+        "max": "1",
+        "binding": {
+          "strength": "required",
+          "valueSet": "https://build.fhir.org/ig/HL7/fhir-us-ndh/ValueSet-QualificationStatusVS.html"
+        }
+      },
+      {
+        "id": "Extension.extension:Status.url",
+        "path": "Extension.extension.url",
+        "fixedUri": "Status"
+      },
+      {
+        "id": "Extension.extension:Status.value[x]",
+        "path": "Extension.extension.value[x]",
+        "type": [
+          {
+            "code": "code"
+          }
+        ],
+        "fixedCode": "active"
+      },
+      {
+        "id": "Extension.extension:Period",
+        "path": "Extension.extension",
+        "sliceName": "Period",
+        "min": 0,
+        "max": "1"
+      },
+      {
+        "id": "Extension.extension:Period.url",
+        "path": "Extension.extension.url",
+        "fixedUri": "Period"
+      },
+      {
+        "id": "Extension.extension:Period.value[x]",
+        "path": "Extension.extension.value[x]",
+        "type": [
+          {
+            "code": "Period"
+          }
+        ]
+      },
+      {
+        "id": "Extension.extension:Wherevalid",
+        "path": "Extension.extension",
+        "sliceName": "Wherevalid",
+        "short": "Where the qualification is valid",
+        "min": 0
+      },
+      {
+        "id": "Extension.extension:Wherevalid.url",
+        "path": "Extension.extension.url",
+        "fixedUri": "Wherevalid"
+      },
+      {
+        "id": "Extension.extension:Wherevalid.value[x]",
+        "path": "Extension.extension.value[x]",
+        "min": 1,
+        "type": [
+          {
+            "code": "CodeableConcept"
+          },
+          {
+            "code": "Reference"
+          }
+        ]
+      },
+      {
+        "id": "Extension.extension:url",
+        "path": "Extension.extension",
+        "sliceName": "url",
+        "short": "http://hl7.org/fhir/us/ndh/StructureDefinition/base-ext-qualification",
+        "min": 1,
```

Comment:
Why is `extension:url` a required field?  I am not even sure what this slice is for...  Would you double-check.  Did you get it from here?  
https://build.fhir.org/ig/HL7/fhir-us-ndh/StructureDefinition-base-ext-qualification.html

Reply:
Yes Matt, This one 
![image](https://github.com/enjoysparkling/signal-mso-fhir-profiles/assets/136109806/a195fa9e-4cf7-4558-835d-ff47d1d508ea)